docs/ViaFoundry/data/data-guide.md

```diff
@@ -0,0 +1,96 @@
+# Getting Your Data on Via Foundry
+
+Welcome to the "Data Guide" section! This is your go-to resource for learning how to efficiently connect and manage your data with Via Foundry. Whether you're integrating data from external sources or uploading local datasets, we've got you covered. In this section, we'll walk you through the various features and methods Via Foundry offers for seamless data integration.
+
+## Connect to your Data in the Cloud
+
+### Connect your storage with Foundry xCloud
+
+Foundry [xCloud](https://www.viascientific.com/products/) enables your cloud storage available in Via Foundry. [xCloud](https://www.viascientific.com/products/) allows organizations and labs to integrate Foundry with their existing cloud infrastructure. Reach out to support to get started, `support@viascientific.com`.
```

Comment:
From @tony-vs 

Move to a different section. Make it clear that this isn't "Connecting" storage. 

xCloud places Foundry in your environment